tools/slims.py

```diff
@@ -243,3 +214,100 @@ def find_runtag_from_fastqs(fastq_objects) -> str:
         raise Exception('No run tag found in fastq objects')
     else:
         return runtag_newest
+
+
+def find_secanalysis_and_state(bioinformatics_record, sample_name) -> tuple[int, str]:
+    """
+    Takes a slims object with bioinformatics content and returns a tuple containing
+    the secondary analysis ID and secondary analysis state
+
+    :param bioinformatics_records: slims bioinformatics object
+    :param sample_name: Sample name (for logging purposes)
+    :return: Tuple with secondary analysis ID and secondary analysis state
+    """
+    # Note, this is a bit hacky since right now a bioinfo object could have multiple secondary analyses.
+    # This means that cntn_cstm_secondaryAnalysis is a list
+    sec_analysis = bioinformatics_record.cntn_cstm_secondaryAnalysis.value[0]
```

Comment:
will RNA-seq pipe always be the first item in the list of secondary analyses? or I guess it is just written like this for now and will be updated later to find correct secondary analysis in the list

Reply:
As it is now, bioinformatics objects can only have a single secondary analysis associated with them. This is just done in order to convert a list to a string. 
In the DNA-object, multiple secanalysis pipes are allowed. This is taken care of in the `slims_records_from_sec_analysis` function.